csm/core/blogic/const.py

```diff
@@ -90,6 +90,9 @@
 # File Collector
 BUNDLE_FILE = 'files.tgz'
 
+# Security
+CERT_TIME_FORMAT = "%Y-%m-%d %H:%M:%S %Z%z"
```

Comment:
i think we dont need "%z" 
output as 
>>> local_time.strftime("%Y-%m-%d %H:%M:%S %Z%z")
'2020-10-27 22:02:12 IST+0530'
>>> local_time.strftime("%Y-%m-%d %H:%M:%S %Z")
**'2020-10-27 22:02:12 IST'**

only IST should be enough

Reply:
Yes, removed %z